sw/airborne/modules/loggers/sd_logger_spi_direct.h

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2015 Bart Slinger <bartslinger@gmail.com>
+ *
+ * This file is part of paparazzi.
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, write to
+ * the Free Software Foundation, 59 Temple Place - Suite 330,
+ * Boston, MA 02111-1307, USA.
+ *
+ */
+
+/** @file modules/loggers/sd_logger.h
+ *  @brief Module for real time logging using an SD Card in SPI mode.
+ */
+
+#ifndef SD_LOGGER_H_
+#define SD_LOGGER_H_
+
+#include "peripherals/sdcard_spi.h"
+
+#define SD_LOGGER_BUFFER_OFFSET 1           /**< Byte reserved for the data token of a multiwrite packet, only applicable to output buffer */
+#define SD_LOGGER_PACKET_SIZE 52            /**< Number of bytes in the LogPacket struct */
```

Comment:
would make sense to use sizeof(struct LogPacket) instead of counting and defining it by hand....